test/com/google/enterprise/adaptor/documentum/DocumentumAdaptorTest.java

```diff
@@ -2534,77 +2541,77 @@ private void testExcludeMetadata(TreeMultimap<String, String> attrs,
-    attributes.put("attr1", "value3");
-    assertEquals(1, attributes.keySet().size());
-    assertEquals(3, attributes.get("attr1").size());
-    TreeMultimap<String, String> expected = TreeMultimap.create(attributes);
-    expected.put("r_object_id", DOCUMENT.pad("aaa"));
+    Metadata attributes = new Metadata();
+    attributes.add("attr1", "value1");
+    attributes.add("attr1", "value2");
+    attributes.add("attr1", "value3");
+    assertEquals(1, attributes.getKeys().size());
+    assertEquals(3, attributes.getAllValues("attr1").size());
+    Metadata expected = new Metadata(attributes);
+    expected.add("r_object_id", DOCUMENT.pad("aaa"));
     testMetadata(attributes, expected);
   }
 
   @Test
   public void testEmptyValueMetadata() throws Exception {
-    TreeMultimap<String, String> attributes = TreeMultimap.create();
-    attributes.put("attr1", "value1");
-    attributes.put("attr2", "value2");
-    attributes.put("attr2", "");
-    attributes.put("attr3", "");
-    TreeMultimap<String, String> expected = TreeMultimap.create(attributes);
-    expected.put("r_object_id", DOCUMENT.pad("aaa"));
+    Metadata attributes = new Metadata();
+    attributes.add("attr1", "value1");
+    attributes.add("attr2", "value2");
+    attributes.add("attr2", "");
+    attributes.add("attr3", "");
+    Metadata expected = new Metadata(attributes);
+    expected.add("r_object_id", DOCUMENT.pad("aaa"));
     testMetadata(attributes, expected);
   }
 
   @Test
   public void testExcludeAttrMetadata() throws Exception {
-    TreeMultimap<String, String> attributes = TreeMultimap.create();
-    attributes.put("attr1", "value1");
-    attributes.put("attr2", "value2");
-    attributes.put("attr3", "value3");
-    attributes.put("foo", "foo1");
-    attributes.put("bar", "bar1");
-    TreeMultimap<String, String> expected = TreeMultimap.create(attributes);
+    Metadata attributes = new Metadata();
+    attributes.add("attr1", "value1");
+    attributes.add("attr2", "value2");
+    attributes.add("attr3", "value3");
+    attributes.add("foo", "foo1");
+    attributes.add("bar", "bar1");
+    Metadata expected = new Metadata();
+    expected.add("attr1", "value1");
+    expected.add("attr2", "value2");
+    expected.add("attr3", "value3");
     String excluded = "foo, bar, r_object_id";
-    expected.removeAll("foo");
-    expected.removeAll("bar");
     testExcludeMetadata(attributes, excluded, expected);
   }
 
   @Test
   public void testObjectTypeMetadata() throws Exception {
-    TreeMultimap<String, String> attributes = TreeMultimap.create();
-    attributes.put("r_object_type", "dm_document");
-    attributes.put("attr2", "value2");
-    TreeMultimap<String, String> expected = TreeMultimap.create(attributes);
-    expected.put("r_object_id", DOCUMENT.pad("aaa"));
-    expected.removeAll("r_object_type");
-    expected.put("r_object_type", "dm_document");
-    expected.put("r_object_type", "dm_sysobject");
+    Metadata attributes = new Metadata();
```

Comment:
Please pick a consistent style. You did one thing in the method above (constructed two Metadata objects from scratch), and another here (copying and modifying the first Metadata object).

Reply:
ping

Reply:
Changed as discussed.